hadoop-ozone/dist/src/main/compose/ozonesecure/docker-compose.yaml

```diff
@@ -48,6 +48,18 @@ services:
     environment:
       KERBEROS_KEYTABS: dn HTTP
       OZONE_OPTS:
+  extradn:
```

Comment:
Thanks for the fix @adoroszlai 

One question here: do really need this `extradn`. I have mixed feelings as this docker-compose environment is used not only for testing but as an example environment. It may be confusing for the users who would like to see a simple secure cluster, that we have this `extradn`.

As far as I see, when you do a `docker-compose scale datanode=2` and `docker-compose scale datanode=3`, the original `/data` volume is recreated, so replication could be tested.

Or (as an alternative, but may be a more complex approach) we can improve the `ClosedContainerReplication` freon test to use certificates...

Reply:
The logic changes look good to me, but I agree with @elek that it would be better to avoid the `extradn` if possible.

One other possible idea - start a cluster with 4 DNs. Create a container and identify one of the DNs which has the container. Then decommission the host and wait for decommission to complete. For decommission to complete, replication must happen, otherwise decommission will hang forever.

Reply:
Thanks @elek and @sodonnel for the review.  I wasn't happy about adding `extradn`, either.  Thanks for the idea about scaling down then up.  The only issue is that it's harder to state our expectation about where the container should be located if the original and new datanodes have the same name.